connectors-api/src/main/ruby/connectors_logger.rb

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+
+require 'java'
+require 'app_config' unless defined?(Rails)
```

Comment:
This is currently failing in the tests, because I can't get the load path to include the maven dependencies